hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/ShowColumnStatsOverlapProcedure.scala

```diff
@@ -258,7 +258,7 @@ class ShowColumnStatsOverlapProcedure extends BaseProcedure with ProcedureBuilde
     val basePath = getBasePath(table)
     val metaClient = createMetaClient(jsc, basePath)
     val storage = metaClient.getStorage
-    val globPath = s"$basePath/*/*/*"
+    val globPath = s"$basePath/**"
```

Comment:
why do we need this change in the context of this patch?

Reply:
The listing below does not return any files in the test so the test will fail. The table is not guaranteed to have this 3 level structure so I am assuming the intention is just to list all files.

Reply:
got it.